grpc/README.md

```diff
@@ -0,0 +1,34 @@
+# GRPC Example
+
+**Note:** This is an advanced scenario useful for people that want to *manually* instrument their own code.
+
+This is a simple example that demonstrates how to use the OpenTelemetry SDK
+to *manually* instrument a simple GRPC based Client/Server application.
+The example utilize the [opentelemetry-grpc-1.6](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/grpc-1.6/library#library-instrumentation-for-grpc-160)
+to instrument both the GRPC client and server. 
+
+# How to run
+
+## Prerequisites
+* Java 1.8.231
```

Comment:
That's oddly specific. Can it run with later versions like 11 or 17 as well?

Reply:
I copied this from other examples, seems it's out of date and I am running on java17. Have it removed!